definitions/GroupFinderCategory.dbd

```diff
@@ -1,15 +1,12 @@
 COLUMNS
-uint field_3184919908?
 locstring name_lang
-uint field_1299825360?
 int ID
 uint flags
 uint orderIndex
 
 BUILD 6.0.1.18179
-field_3184919908<32>
```

Comment:
These defs come from DBMeta. Are you really sure they had the wrong column count?! Are you sure you have checked with 18179?

Reply:
Yeah this is wrong. When I originally pulled the DBs from Martin's site any that failed I got the previous build.

Basically everything I've done is crap and needs redoing.